### PR TITLE
rdf-prefix-update.pl to update the list; update it as of today

### PR DESCRIPTION
**Pull request by VladimirAlexiev**
TODO: reimplement perl script as elisp function rdf-prefix-update

### CODE REVIEW

File: rdf-prefix.el [hunk 1]
```diff
@@ -2596,7 +2863,8 @@
     ("zem" . "http://s.zemanta.com/ns#")
     ("zoology" . "http://purl.org/NET/biol/zoology#")
     ("zoomaterms" . "http://rdf.ebi.ac.uk/vocabulary/zooma/")
-    ("zr" . "http://explain.z3950.org/dtd/2.0/")))
+    ("zr" . "http://explain.z3950.org/dtd/2.0/")
+))
```
Comment (simenheg) [inline rdf-prefix.el]: Could you update the script to keep the previous format with the closing parentheses on the same line?

File: README.md [hunk 2]
```diff
@@ -23,4 +23,7 @@ Alternatively, it can be installed via [MELPA](https://melpa.org/).
 This package provides the `rdf-prefix-insert` command, which is used to look
 up- and insert URIs based on a prefix.
 
-The prefix list is pulled regularly from http://prefix.cc/.
+The prefix list is pulled semi-regularly from http://prefix.cc/.
```
Comment (simenheg) [inline README.md:26]: True... ;)

**Review by simenheg — COMMENTED** [review]
Cool, @VladimirAlexiev! Just a tiny request, otherwise this looks good to me.

File: rdf-prefix.el (hunk 1 above)
Comment (VladimirAlexiev) [reply rdf-prefix.el]: @simenheg 
done, see https://github.com/simenheg/rdf-prefix/pull/2/commits/fb274f74048da15bf9bd3037339436d87ae260d3.
Cheers!

**Pull request merged**